stream-chat-android/src/main/java/com/getstream/sdk/chat/view/EndlessScrollListener.kt

```diff
@@ -3,12 +3,12 @@ package com.getstream.sdk.chat.view
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
 
-internal class EndlessScrollListener(
```

Comment:
I had to change the visibility of some classes and accessors in many classes in other to clone the MessageListView.  It is big change, but I don't see a way to clone MessageListView without doing that or make a super huge PR that brings a lot of classes to the new module